src/pkg/packager/common.go

```diff
@@ -342,41 +341,13 @@ func (p *Packager) archivePackage(destinationTarball string) error {
 
 	// If a chunk size was specified and the package is larger than the chunk size, split it into chunks.
 	if p.cfg.CreateOpts.MaxPackageSizeMB > 0 && fi.Size() > int64(chunkSize) {
-		spinner.Updatef("Package is larger than %dMB, splitting into multiple files", p.cfg.CreateOpts.MaxPackageSizeMB)
-		chunks, sha256sum, err := utils.SplitFile(destinationTarball, chunkSize)
-		if err != nil {
-			return fmt.Errorf("unable to split the package archive into multiple files: %w", err)
-		}
-		if len(chunks) > 999 {
+		if fi.Size() / int64(chunkSize) > 999 {
 			return fmt.Errorf("unable to split the package archive into multiple files: must be less than 1,000 files")
 		}
-
-		status := fmt.Sprintf("Package split into %d files, original sha256sum is %s", len(chunks)+1, sha256sum)
-		spinner.Updatef(status)
-		message.Debug(status)
-		_ = os.RemoveAll(destinationTarball)
-
-		// Marshal the data into a json file.
-		jsonData, err := json.Marshal(types.ZarfSplitPackageData{
-			Count:     len(chunks),
-			Bytes:     fi.Size(),
-			Sha256Sum: sha256sum,
-		})
+		spinner.Updatef("Package is larger than %dMB, splitting into multiple files", p.cfg.CreateOpts.MaxPackageSizeMB)
+		_, _, err := utils.SplitFile(destinationTarball, chunkSize)
```

Comment:
Since the return variables are not used the function signature might as well not return them and only return `err`.  Also since this reduces messaging IMO it would be good to implement the progressbar interface for this function where `utils.SplitFile` takes in an `io.Writer` called `progressWriter` and this caller instantiates a `message.NewProgressBar` taking in `fi.Size()` as the total number of bytes (when written to the progress bar will progress forward giving more indication of how long the process is going to take than a simple spinner - before it would show the file it was currently on in the spinner and a progressbar would bring that show of progress back and more).

Reply:
If you have questions on how to implement the progressbar interface let us know and we can help.

For reference you can see how this progressbar is used with io.Copy:

https://github.com/defenseunicorns/zarf/blob/23618d95c49f177d022fb45c206530b6dece54cc/src/pkg/oci/copier.go#L47
https://github.com/defenseunicorns/zarf/blob/23618d95c49f177d022fb45c206530b6dece54cc/src/pkg/oci/copier.go#L100

Reply:
Sure I can look into adding that in when I get some free time. I'm hoping to go a step further if we are open to using archiver v4. It looks like we could use it to split the file as it gets archived and save some time

Reply:
However, if we go that route we likely won't be able to do a progressbar as we won't know the compressed file size ahead of time.

Reply:
@Racer159, I've cleaned up the returns and have implemented the progressbar. I opted for ending the spinner after the initial archive is written, displaying a note that we are going to split the archive, and then doing a progressbar for the split.

Although this is after it has finished and you won't be able to actually see the progressbar, it will look something like this
<img width="1454" alt="image" src="https://github.com/defenseunicorns/zarf/assets/114503075/706f46c6-85b9-4bd8-86e1-72720619e839">

Reply:
Sorry for the delay - I think that works for the progressbar - it will go quick on smaller packages but looks decent on bigger ones where `--max-package-size` is more likely to be used.

![image](https://github.com/defenseunicorns/zarf/assets/3977569/166c2bc0-12e9-47ba-b094-a70a25a159f3)
![image](https://github.com/defenseunicorns/zarf/assets/3977569/2136fd87-1467-498d-8146-64e0aafe5e27)